refactor(optimize-load/unleash): code splitting + lazy mount

crystalcheong opened the pull request:
- [reduce time to load stats on homepage](https://www.notion.so/switcheo/reduce-time-to-load-stats-on-homepage-d9c00dad728c48d994c0836d030d7b50?pvs=4)

sarah-thong commented on src/@demex-info/views/UnleashTrader/components/MarketsMarquee.tsx:
```diff
@@ -255,18 +215,6 @@ const useStyles = makeStyles((theme) => ({
-  },
   marketsCard: {
     [theme.breakpoints.down("sm")]: {
       padding: "0.75rem 1rem",
```
Heyyo, no issues with this code. Just that it would be good to optimise the CDP queries as well. 

Atm, all the CDP queries are originating from this SDK function `getModuleTotalCollateralUsdVal`, though the queries are not very efficient (e.g. querying supply for each individual CDP asset rather than querying all in one query). Could you try and optimise the queries a bit? Thank you!
<img width="638" alt="Screenshot 2023-06-09 at 1 42 57 PM" src="https://github.com/Switcheo/demex-website/assets/65583522/c741af92-c251-44d9-9801-53c6db05ca40">

crystalcheong replied:
Updated the `getModuleTotalCollateralUsdVal` in https://github.com/Switcheo/carbon-js-sdk/pull/333

sarah-thong commented on src/@demex-info/views/UnleashTrader/components/MarketSparkline.tsx:
```diff
@@ -0,0 +1,96 @@
+          from: new Long(Math.floor(new Date(new Date().setDate(new Date().getDate() - 7)).getTime() / 1000)).toNumber(),
+          to: new Long(Math.floor(new Date().getTime() / 1000)).toNumber(),
+        };
+        const response = await ws.request<{ result: WSModels.Candlestick[] }>(WSConnectorTypes.WSRequest.Candlesticks, params) as WSResult<{ result: WSModels.Candlestick[] }>;
```
So sorry, I forgot on v2.26.0, ws get requests will be removed. As such, will need to revert back to GRPC queries.

Other than that, your approach looks correct. Thanks!

crystalcheong replied:
reverted in [cd06cba](https://github.com/Switcheo/demex-website/pull/117/commits/cd06cbabc25ff74058f6f3c6ad685e792dfb6b55)

sarah-thong commented:
Also, merged + published your SDK branch in alr: `v0.4.31`, can upgrade when you're back in office.

crystalcheong commented:
> Merged + published your SDK branch in alr: `v0.4.31`, can upgrade when you start work again.
> 
> Also, could you update the token svgs as well? <img alt="Screenshot 2023-06-09 at 6 51 11 PM" width="433" src="https://user-images.githubusercontent.com/65583522/244672632-9d0e081c-d967-4849-91cf-b1d43d58016a.png">

<br/>

addressed in [fff5527](https://github.com/Switcheo/demex-website/pull/117/commits/fff5527dc023fe5cc32a533edd060d2fc81284db):
- [X] upgrade sdk to `v0.4.31`
- [X] update icons
     > <img width="137" alt="image" src="https://github.com/Switcheo/demex-website/assets/65748007/63341dad-8d80-48c7-95c8-53a1b0eecf66">
<details open>
<summary>🚨 notable changes</summary>

  - `importAssets` to dynamically import all assets in the specified folder
  - added `@types/webpack-env` dependency for `NodeRequire` (used in `importAssets`)
</details>
